konnectd/ui/build/static/js/0.d0436ff0.chunk.js

```diff
@@ -0,0 +1,3 @@
utAdornedEnd:{},inputHiddenLabel:{}}}),{name:"MuiInputBase"})(x)},53:function(e,t,n){"use strict";t.a=function(e){return e&&e.ownerDocument||document}},54:function(e,t,n){"use strict";Object.defineProperty(t,"__esModule",{value:!0});t.default=function(e){return function(t){return e?"function"===typeof t?t():t:null}},e.exports=t.default},57:function(e,t,n){"use strict";function r(e){var t=e.props,n=e.states,r=e.muiFormControl;return n.reduce((function(e,n){return e[n]=t[n],r&&"undefined"===typeof t[n]&&(e[n]=r[n]),e}),{})}n.d(t,"a",(function(){return r}))},58:function(e,t,n){"use strict";n.d(t,"a",(function(){return i}));var r=n(7),o=n.n(r),a=n(111);function i(){return o.a.useContext(a.a)}},68:function(e,t,n){"use strict";function r(e,t){"function"===typeof e?e(t):e&&(e.current=t)}n.d(t,"a",(function(){return r}))},72:function(e,t,n){"use strict";var r=n(9),o=n(7),a=n.n(o),i=n(10),c=(n(8),n(11)),s=n(12),l=n(17),u=a.a.forwardRef((function(e,t){var n=e.children,o=e.classes,s=e.className,u=e.color,d=void 0===u?"inherit":u,p=e.component,f=void 0===p?"svg":p,m=e.fontSize,h=void 0===m?"default":m,b=e.htmlColor,v=e.titleAccess,g=e.viewBox,y=void 0===g?"0 0 24 24":g,x=Object(i.a)(e,["children","classes","className","color","component","fontSize","htmlColor","titleAccess","viewBox"]);return a.a.createElement(f,Object(r.a)({className:Object(c.a)(o.root,s,"inherit"!==d&&o["color".concat(Object(l.a)(d))],"default"!==h&&o["fontSize".concat(Object(l.a)(h))]),focusable:"false",viewBox:y,color:b,"aria-hidden":v?"false":"true",role:v?"img":"presentation",ref:t},x),n,v?a.a.createElement("title",null,v):null)}));u.muiName="SvgIcon";var d=Object(s.a)((function(e){return{root:{userSelect:"none",width:"1em",height:"1em",display:"inline-block",fill:"currentColor",flexShrink:0,fontSize:e.typography.pxToRem(24),transition:e.transitions.create("fill",{duration:e.transitions.duration.shorter})},colorPrimary:{color:e.palette.primary.main},colorSecondary:{color:e.palette.secondary.main},colorAction:{color:e.palette.action.active},colorError:{color:e.palette.error.main},colorDisabled:{color:e.palette.action.disabled},fontSizeInherit:{fontSize:"inherit"},fontSizeSmall:{fontSize:e.typography.pxToRem(20)},fontSizeLarge:{fontSize:e.typography.pxToRem(35)}}}),{name:"MuiSvgIcon"})(u);function p(e,t){var n=a.a.memo(a.a.forwardRef((function(t,n){return a.a.createElement(d,Object(r.a)({},t,{ref:n}),e)})));return n.muiName=d.muiName,n}n.d(t,"a",(function(){return p}))},79:function(e,t,n){e.exports=n(323)},80:function(e,t,n){"use strict";var r=n(7),o=n.n(r).a.createContext({});t.a=o},82:function(e,t,n){"use strict";function r(e){return function(e){if(Array.isArray(e)){for(var t=0,n=new Array(e.length);t<e.length;t++)n[t]=e[t];return n}}(e)||function(e){if(Symbol.iterator in Object(e)||"[object Arguments]"===Object.prototype.toString.call(e))return Array.from(e)}(e)||function(){throw new TypeError("Invalid attempt to spread non-iterable instance")}()}n.d(t,"a",(function(){return r}))}}]);
```

Comment:
Codacy found an issue: [The numeric literal '.01' will have at different value at runtime.](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6337760)